war/src/main/webapp/scripts/hudson-behavior.js

```diff
@@ -227,27 +227,31 @@ var FormChecker = {
    *
    * @param url
    *      Remote doXYZ URL that performs the check. Query string should include the field value.
-   * @param method
-   *      HTTP method. GET or POST. I haven't confirmed specifics, but some browsers seem to cache GET requests.
    * @param target
    *      HTML element whose innerHTML will be overwritten when the check is completed.
    */
-  delayedCheck: function (url, method, target) {
-    if (url == null || method == null || target == null) {
-      // don't know whether we should throw an exception or ignore this. some broken plugins have illegal parameters
-      return;
+  delayedCheck: function (url, target) {
```

Comment:
This is a breaking change for any current callers of this method (e.g., `matrix-auth`).

Reply:
this was adjusted in the separated PR which has been merged, I'll rebase against master and it'll be sorted